eng/versioning/version_client.txt

```diff
@@ -19,9 +19,9 @@ com.azure:azure-core-serializer-json-gson;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-core-serializer-json-jackson;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-core-test;1.4.0;1.5.0-beta.1
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.6;1.0.0-beta.7
-com.azure:azure-cosmos;4.3.1;4.3.2-beta.3
-com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.0.1-beta.1
-com.azure:azure-cosmos-encryption;1.0.0-beta.1;1.0.0-beta.1
+com.azure:azure-cosmos;4.3.1;4.4.0-beta.1
+com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.1.0-beta.1
+com.azure:azure-cosmos-encryption;1.1.0-beta.1;1.1.0-beta.1
```

Comment:
benchmark and encryption should not be incremented. Nor should be spring-data-cosmos modules. 
you can even skip azure-cosmos version increment here, and just do it one time when we are ready to release.

Reply:
Fixed